src/utils/string/StringUtils.js

```diff
@@ -11,9 +12,10 @@ import TextBold from '../../components/Text/TextBold';
 import TopicText from '../../components/TopicText';
 // eslint-disable-next-line import/no-cycle
 import removePrefixTopic from '../topics/removePrefixTopic';
+import {COLORS} from '../theme';
+import {DEFAULT_PROFILE_PIC_PATH, DEFAULT_TOPIC_PIC_PATH} from '../constants';
 import {getAnonymousUserId} from '../users';
 import {getUserId} from '../token';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been rearranged. While this doesn't affect the functionality, it's generally a good practice to keep imports organized. Typically, absolute imports are placed before relative imports and they are sorted alphabetically. However, if there is no specific convention followed in your project, you can ignore this comment.
```diff
- import {COLORS} from '../theme';
- import {DEFAULT_PROFILE_PIC_PATH, DEFAULT_TOPIC_PIC_PATH} from '../constants';
- import {getAnonymousUserId} from '../users';
- import {getUserId} from '../token';
+ import {getAnonymousUserId} from '../users';
+ import {getUserId} from '../token';
+ import {COLORS} from '../theme';
+ import {DEFAULT_PROFILE_PIC_PATH, DEFAULT_TOPIC_PIC_PATH} from '../constants';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->